projects/tad/index.md

```diff
@@ -0,0 +1,84 @@
+<!-- markdownlint-disable MD013 -->
+# Transparency of algorithmic decision making
+
+This document contains a checklist with requirements for tools we could use to help with the transparency of algorithmic decision making.
+
+The requirements are based on:
+
+- **ISO 25010 standard:** This standard defines the eight quality characteristics and provides a framework for evaluating software quality.
```

Comment:
What do you think of adding references to relevant sources we have used?

For example:
```suggestion
- **ISO 25010 standard:** This standard defines the eight quality characteristics and provides a framework for evaluating software quality ([source](https://www.iso.org/standard/35733.html)).
```